src/components/JobCard/CardStyles.tsx

```diff
@@ -14,18 +14,20 @@ export default StyleSheet.create({
   },
```

Comment:
lol couldn't find a good place to put this comment. i might've missed but do you use all four of the heart pngs. if not can you delete the unused ones.